### ISSUE TITLE
Slow start and interface problems (scrolling, cursor)

### ISSUE THREAD

**Issue by burkingman** [mltframework/shotcut#865]
Using Shotcut 20.04.12 (AppImage version) on Linux Mint 19.3.
I've been using the AppImage downloads exclusively. Noticed problems since version 19.02.28; version 19.01.27 seemed OK; haven't tried all versions in between.

1) From about 19.02.28 onwards, Shotcut has been a little slow to start. The Shotcut window's title bar appears but my desktop is still visible, though slightly shifted, and it takes a little while before the Shotcut interface appears. (The delay isn't that long, not a big problem, but I'm mentioning it in case it helps with diagnostics.)
2) There are some interface glitches. The worst is that the horizontal scrollbar below the timeline doesn't work: I can't scroll, whether I click on the scrollbar or on the arrows to either side, or try to drag the slider, or use the mouse wheel. If I click on the scrollbar, Shotcut reacts as if I've clicked on the timeline instead. Similar problem with certain filters with a wide interface (notably the color grading filter). Also: the cursor naturally changes when passing over certain parts of the interface, but sometimes doesn't change back to normal afterwards, or takes a while to change back.

The scrolling problem in particular makes Shotcut hard to use. I suspect the problem may coincide with Shotcut upgrading to QT 5.9.7, but you would know better than I would.

I'm attaching the output produced by Shotcut upon starting (without the long list of formats and codecs).

[shotcut_200412.log](https://github.com/mltframework/shotcut/files/4603977/shotcut_200412.log)

**Comment by ddennedy** [comment]
It is probably due to the Qt upgrade and is perhaps related to the OpenGL on your system or some other weird system specific interaction. Two things to try: The flatpak uses Qt 5.14 or in the current version try with Settings > Display Method > Software.

**Comment by burkingman** [comment]
Thanks for the quick answer! Changing the display method didn't work. Couldn't find anything wrong with my OpenGL but I can't rule out system issues. Will probably try the flatpak soon and report back. (Extra info: I tried restoring the default layout and the timeline area winds up too low, I can barely see the top of it. Could be that it's a height calculation issue more than a scrollbar issue.)

**Comment by ddennedy** [comment]
I made a change to improve the initial height of the timeline.

**Comment by burkingman** [comment]
Found a workaround for the interface issue (part 2 of my original post)! Most of the time, I use a Wacom tablet instead of a mouse; it basically works great in all my regular applications. Switched from tablet to mouse recently and the problem in Shotcut goes away: the cursor behaves as it should, and so do the scrollbars.

I'm now using the latest AppImage. It's still slow to start, but that's no big deal. Looks like it'll be okay as long as I use it with the mouse.

(Sorry I didn't test the flatpak as planned: changed my mind when I saw the install size.)
